src/main/java/noobconversation/Conversation.java

```diff
@@ -0,0 +1,74 @@
+package noobconversation;
+
+import format.Formatting;
+import task.Task;
+
+import java.util.Scanner;
+
+public class Conversation {
+
+    protected Formatting f;
+
+    public Conversation(){
+        f = new Formatting();
+    }
+
+    public void communicate() {
+        CommunicateCaseHandle caseHandle = new CommunicateCaseHandle();
+        Scanner in = new Scanner(System.in);
+
+        Task[] list = new Task[100];
+        int index = 0;
+        String line = in.nextLine().trim();
+
+        while (!line.equalsIgnoreCase("bye")) {
+            if (line.equalsIgnoreCase("list")) {
+                caseHandle.listHandle(index, list);
+            } else if (line.toLowerCase().startsWith("unmark")) {
+                caseHandle.unmarkHandle(line, in, index, list);
+            } else if (line.toLowerCase().startsWith("mark")) {
+                caseHandle.markHandle(line, in, index, list);
+            } else {
+                if(caseHandle.taskHandle(line, in, list, index) == 66319){
+                    index++;
+                }
+            }
+            line = in.nextLine().trim();
+        }
+    }
+
+    public void describeFunctionality() {
```

Comment:
Instead, describe what the code does and maybe use `printFeatures()` or `printFunctionality()` instead.